library/src/css.d.ts

```diff
@@ -0,0 +1,7 @@
+declare module '*.css' {
+  interface IClassNames {
+    [className: string]: string;
+  }
+  const classNames: IClassNames;
+  export = classNames;
+}
```

Comment:
Need to check where this comes from and why I need it :)

Reply:
Typescript is complaining about `import styles from './styles.css';` imports, with the type definition file you remove the errors.